### DISCUSSION TITLE
Wapro ERP

### DISCUSSION THREAD

**Discussion by waran70** [YetiForceCompany/YetiForceCRM#16239] [General] [answered]
What is the point of doing such exotic integration like Wapro and publishing it for everybody?
What if I order from you some Nigerian integration with local goat farmer? Will you add it to yetiforce too?
Are Yetiforce truly designated for international market or Polish and Ukrainian only?

**Comment by mariuszkrzaczkowski** [comment]
> What is the point of doing such exotic integration like Wapro and publishing it for everybody? 

Is it wrong that we are expanding the system with new integrations? separating it from the system is more difficult to maintain.
> What if I order from you some Nigerian integration with local goat farmer? 

I don't know, it depends on the integration itself if it will be useful to others. If others will be able to use it, you can add.
> Will you add it to yetiforce too? Are Yetiforce truly designated for international market or Polish and Ukrainian only?

You can see that you are biased, we are creating a free system for everyone.
We have twice as many registrations in the United States than in Poland, despite the fact that we do not advertise anywhere.

**Comment by waran70** [comment, marked as answer]
> Is it wrong that we are expanding the system with new integrations? separating it from the system is more difficult to maintain.
It is not wrong to keep developing, wrong is when you are bloating base code, and this is what confusing professional programmers who will be kept away from uncertain stability

> I don't know, it depends on the integration itself if it will be useful to others. If others will be able to use it, you can add.
Others here doesn't meant statistically whole user base. All people here want a predictable set of modules and functionality, not some sort of whoever order we all will get it- pointless. This is what makes system complicated. It frighten me to think what it gonna be in next 3 years

> You can see that you are biased, we are creating a free system for everyone.
I am totally against bias and against separating users on the basis of nationality. That's why when part of development (or not too wise announcements) is focused ONLY on certain geographic zone, it rings my bell.

Advice: 
1. Cut out controlling users by weird registrations - it hardly works for you anyway.
2. Disintegrate Marketplace from system - it looks and feel weird. Keep some link - it will be definitely enough
3. Focus on modules which are common out of the box in competitive systems. Rest throw to marketplace
4. Do things to make this place friendly for developers who may bring own extensions (But not vtiger way - they sucks). Analyze how Ubuntu or Libre Office get fed.

**Reply by apcloic** [reply]
@waran70 So you mark as answer your own reply to your own question ?

**Reply by waran70** [reply]
Yp, to close the thread before fire starts.
